modules/solid_mechanics/src/materials/SymmIsotropicElasticityTensor.C

```diff
@@ -68,6 +68,15 @@ SymmIsotropicElasticityTensor::shearModulus() const
 }
 
 Real
+SymmIsotropicElasticityTensor::youngsModulus() const
+{
+  if (!_E_set)
+    mooseError("Youngs modulus not set");
+
+  return _E;
+}
```

Comment:
The material model which uses this function is in Bison. So I am not sure how to test this.